Cargo.toml

```
@@ -55,7 +55,7 @@ path = "examples/internal/rbspy-connect.rs"
 thiserror ="1.0"
 log = "0.4"
 names = "0.13.0"
-reqwest = { version = "0.11", features = ["blocking", "rustls-tls-native-roots"]}
+reqwest = { version = "0.11", features = ["blocking"], default-features = false }
```

```suggestion
reqwest = { version = "0.11", features = ["blocking", "default-tls"], default-features = false }
```
I think with `default-features = false` it does not compile tls support at all.

---

What about always including the webpki roots? This way there will always be some tls support without any external dynamic libraries that a user must install onto their system?

---

I like the idea to enable rustls by default for all our binaries.
We still need to have a feature flag to enable system ssl in case someone want it(for security reasons or smth else)

So I think we need two features "rustls-tls", "native-tls", and former can be used by default

---

notice you are probably still going to need libraries, for example libunwind

```
➜  pyroscope_cli git:(3098ca6) ✗ ldd target/debug/pyroscope-cli 
        linux-vdso.so.1 (0x00007fffb5cd7000)
        libunwind.so.8 => /lib/x86_64-linux-gnu/libunwind.so.8 (0x00007fe1e25c4000)
        libunwind-ptrace.so.0 => /lib/x86_64-linux-gnu/libunwind-ptrace.so.0 (0x00007fe1e25be000)
        libunwind-x86_64.so.8 => /lib/x86_64-linux-gnu/libunwind-x86_64.so.8 (0x00007fe1e25a1000)
        libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007fe1e24ba000)
        libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007fe1e2292000)
        /lib64/ld-linux-x86-64.so.2 (0x00007fe1e490e000)
        liblzma.so.5 => /lib/x86_64-linux-gnu/liblzma.so.5 (0x00007fe1e2265000)

```